graphics/managed_surface.h

```diff
@@ -30,143 +30,159 @@
 
 namespace Graphics {
 
+/**
+ * @defgroup graphics_managed_surface Managed surface
+ * @ingroup graphics
+ *
+ * @brief The ManagedSurface class that adds extra functionality on top of the Surface class.
+ *
+ * @{
+ */
+
 class Font;
 
 /**
- * A derived graphics surface, which handles automatically managing the allocated
- * surface data block, as well as introducing several new blitting methods
+ * A derived graphics surface, which supports automatically managing the allocated
+ * surface data block and introduces several new blitting methods.
  */
 class ManagedSurface {
+	/** See @ref Font. */
 	friend class Font;
 private:
 	/**
-	 * The Graphics::Surface that the managed surface encapsulates
+	 * The Graphics::Surface that the managed surface encapsulates.
 	 */
 	Surface _innerSurface;
 
 	/**
 	 * If set, the inner surface will be freed when the surface is recreated,
-	 * as well as when the surface is destroyed
+	 * as well as when the surface is destroyed.
 	 */
 	DisposeAfterUse::Flag _disposeAfterUse;
 
 	/**
-	 * Stores the owning surface if this If this managed surface represents
-	 * a sub-section of another
+	 * If this managed surface represents a subsection of another managed surface,
+	 * store the owning surface.
 	 */
 	ManagedSurface *_owner;
 
 	/**
-	 * For sub-section areas of an owning parent managed surface, this represents
-	 * the offset from the parent's top-left corner this sub-surface starts at
+	 * For a managed surface that has a parent, this represents the offset from
+	 * the parent's top-left corner where this sub-surface starts at.
 	 */
 	Common::Point _offsetFromOwner;
 
 	/**
-	 * Automatic transparency color. When set, doesn't require transparency to be
-	 * explicitly passed, and blit methods pass on to transBlit
+	 * Automatic transparency color. When set, it does not require transparency to be
+	 * explicitly passed, and blit methods pass on to transBlit.
 	 */
 	uint _transparentColor;
 	bool _transparentColorSet;
 
 	/**
-	 * Local palette for 8-bit images
+	 * Local palette for 8-bit images.
 	 */
 	uint32 _palette[256];
 	bool _paletteSet;
 protected:
 	/**
-	 * Base method that descendent classes can override for recording affected
-	 * dirty areas of the surface
+	 * Base method that descendant classes can override for recording the affected
+	 * dirty areas of the surface.
 	 */
 	virtual void addDirtyRect(const Common::Rect &r);
 
 	/**
-	 * Inner method for blitting
+	 * Inner method for blitting.
 	 */
 	void blitFromInner(const Surface &src, const Common::Rect &srcRect,
 		const Common::Point &destPos, const uint32 *palette);
 
 	/**
-	 * Inner method for copying another surface into this one at a given destination position
+	 * Inner method for copying another surface into this one at a given destination position.
+	 * @todo there are a lot of parameters here, are they all fairly clear or should we document them?
```

Comment:
I don't think you need to document them. As the description indicates, it is an internal function for the class. Code using a `ManagedSurface` will not use this function directly.